.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,16 @@
+name: Deploy documentation website
+
+on:
+  # Runs on pushes targeting the `main` branch. 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)